utils/_context/_scenarios/__init__.py

```diff
@@ -1584,6 +1584,14 @@ def all_endtoend_scenarios(test_object):
         doc="Scenario with custom headers for DD_TRACE_HEADER_TAGS that libraries should reject",
     )
 
+    # disable_trace_stats = EndToEndScenario(
```

Comment:
You can use the `APPSEC_DISABLED` scenario (don't mind about the name, it should have been named `DISABLE_ALL_FEATURES`, on my procrastination list ...)